gprofiler/exceptions.py

```diff
@@ -28,11 +28,13 @@ def __str__(self):
 
 
 class CalledProcessTimeoutError(CalledProcessError):
-    def __init__(self, timeout: float, returncode: int, cmd: Union[str, List[str]], output=None, stderr=None):
+    def __init__(
+        self, timeout: float, returncode: int, cmd: Union[str, List[str]], output: Any = None, stderr: Any = None
```

Comment:
`output` is always `str`, no? And `stderr` as well

Reply:
They are, but I wanted to match `CalledProcessError`'s `__init__`, and he allows `Any`

Reply:
But if we always pass `str` here, why not require it?